packages/speed.txt

```diff
@@ -1,4 +0,0 @@
-orjson>=3.5.4
```

Comment:
Why are you removing the packages extra other requirements files? There also setup when working on the repo

Reply:
just do pip install .[package]